db/migrate/20160805000209_create_contacts.rb

```diff
@@ -0,0 +1,17 @@
+class CreateContacts < ActiveRecord::Migration
+  def change
+    create_table :contacts do |t|
+      t.string :first_name
+      t.string :last_name
+      t.string :email
+      t.string :phone
+      t.string :service
+      t.date :date
+      t.string :availability
+      t.integer :skillset
+      t.text :notes
+
+      t.timestamps null: false
+    end
+  end
+end
```

Comment:
I think its a good practice to add null: false for things you are going to require. It provides a failing level of check at the database level and is good design, in my opinion. Reinforces the validates_presence_of when you add 'em in. We didn't really discuss what is or isn't required, though. No big deal.